README.md

```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Surver Form
+> This is a project that shows my code review skills using Github Review features
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  A browser
+```
+### Setup
+Clone the repository and change the directory to past-project
+``` 
+  $ git clone git@github.com:Felix45/review-past-project.git
+
+  $ cd past-project
+
+```
+
+### Usage
+Start the http application server
+``` 
+  Open with live server
+```
+
+### Deployment
+- [Visit Application Homepage](http://localhost:8080)
```

Comment:
I think this link provided here will never work. pointing that out would be great.
What do you think? :rocket: 
`hint`:
`The deployment is made on GitHub pages `